bin/pocs_shell

```diff
@@ -148,6 +148,11 @@ class PocsShell(Cmd):
         """Setup and initialize a POCS instance."""
         simulator = listify(arg[0].split())
 
+        # TODO(wtgee) Incorporate real power readings
```

Comment:
This is OK for now, but I was wondering about adding an entry to the config for this, That makes it easy for "optional" features such as this to be enabled or disabled on a per site basis. Thoughts?